Dockerfile

```
@@ -0,0 +1,28 @@
+FROM node:lts-bullseye as build-deps
```

```suggestion
FROM node:8 as build-deps
```
It's helpful to document the required version of node, but including it in the Dockerfile makes it even more explicit (and tracked by git!)

Pinning to a specific version also makes it machine-readable, which allows automated tools (eg [Renovate](https://github.com/renovatebot/renovate), [Dependabot](https://github.com/dependabot)) to audit and even automatically upgrade dependencies

---

Thanks, locked the image, but bumped it to 18 as 8 was giving me issues in the build.